### ISSUE TITLE
[Screen Reader - Intelligent Support - IRIS]: When focus lands on the ‘Did that answer your question or solve your problem?’ Screen reader is not announcing the label name.

### ISSUE THREAD

**Issue by pgopinadhu** [microsoft/BotFramework-WebChat#4552]
**Repro steps:**
Step 1: Open the above URL in a browser. Resource360 Dashboard Page will be displayed. 
Step 2: Navigate to Iris bot icon in the header and invoke it
Step 3: Iris bot chat window will be displayed, Turn on Narrator
Step 4: Type start over-> Top questions for my role-> select Where can I download the Time 2.0 mobile app for Android ->Select no (Thumbs down)-> Did that answer your question or solve your problem?
Step 5: verify whether the screen reader announcing the ‘Did that answer your question or solve your problem?’ or not.

**Actual Result:**
When focus lands on the ‘Did that answer your question or solve your problem?’ Screen reader is not announcing the label name.

**Expected result:**
Screen reader should announce the label name of ‘Did that answer your question or solve your problem?’.

**Note:**
Same issue happens in the ‘Can you rephrase your question and ask again?' also
PFA<MAS3.3.2_screen reader is not announcing the question did that answer your question or solve your problem.mp4>

**Observation:**
Same issue observed with NVDA also.
PFA<MAS3.3.2_Screen reader is not announcing the question 'Can you rephrase your question and ask again'.png>
PFA<MAS3.3.2_screen reader is not announcing the question did that answer your question or solve your problem.png>

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b3752B546-30D8-44D7-8221-6FC32EEB3AC8%7d&file=MAS%203.3.2%20%E2%80%93%20Labels%20or%20Instructions.docx&action=default&mobileredirect=true&cid=7a236812-6a54-421b-a563-b324db5295e5

**Test Environment:**
**OS:** Windows 11 22H2 (OS Build 22621.674)
**Browsers:** Edge Chromium (107.0.1418.26)
**Screen Reader:** Narrator
**URL:** https://servicesuat.microsoft.com/#/opdashboard/

**Comment by pgopinadhu** [comment]
#A11yMAS;#A11ySev2;#HCL;#MAS3.3.2;#Accessibility;#Win11-Edge(Chromium);#BotFrameworkWebChat-Jul22;

**Comment by pgopinadhu** [comment]
#A11yMAS;#A11ySev2;#HCL;#MAS3.3.2;#Accessibility;#Win11-Edge(Chromium);#BotFrameworkWebChat-Jul22;

**Comment by compulim** [comment]
@pgopinadhu I couldn't find the Iris bot in the header. The Resource360 Dashboard appears empty to me.

Can you verify if the Iris bot is running?

**Comment by compulim** [comment]
As we cannot continue to investigate without access to the environment and lacking of response from the bug author, I am closing this bug.

**Comment by pgopinadhu** [comment]
@compulim Product Team Provided the Environment access. Please check and let us know if there are any issues. Thanks!

**Comment by compulim** [comment]
I verified the Iris Bot is running Web Chat 4.15.7.

I am not able to repro the issue. In the recordings below, around 00:19, I can hear "Did that answer your question or solve your problem?"

https://user-images.githubusercontent.com/1622400/224416455-5b6cd992-df2c-44f4-9c43-e96637ecd9ee.mp4
